MetadataProcessor.Shared/Utility/nanoCLR_DataType.cs

```diff
@@ -1,12 +1,16 @@
-//
-// Copyright (c) .NET Foundation and Contributors
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
 // Original work from Oleg Rakhmatulin.
-// See LICENSE file in the project root for full license information.
-//
 
 namespace nanoFramework.Tools.MetadataProcessor
 {
-    public enum nanoCLR_DataType : byte
+    //////////////////////////////////////////////////////////////////////////////////
+    // !!! KEEP IN SYNC WITH enum CLRDataType (in nanoCLR_TypeSystem Debugger) !!! //
+    // !!! KEEP IN SYNC WITH enum CLRDataType (in nanoCLRT_Types.h in CLR)     !!! //
```

Comment:
nothing to keep in sync here?

Reply:
All of these are to be kept in sync as noted in the comment. Or I'm missing your point...